sql/hive/src/test/scala/org/apache/spark/sql/parquet/parquetSuites.scala

```diff
@@ -299,6 +301,37 @@ class ParquetDataSourceOnSourceSuite extends ParquetSourceSuiteBase {
     super.afterAll()
     setConf(SQLConf.PARQUET_USE_DATA_SOURCE_API, originalConf.toString)
   }
+
+  test("insert array into parquet hive table using data source api") {
```

Comment:
`spark.sql.parquet.useDataSourceApi` is turn on already in the unit test I added. It failed on the master I just pulled.